### ISSUE TITLE
about self-training log 

### ISSUE THREAD

**Issue by ShuojieMo** [Nightmare-n/GD-MAE#8]
Could you provide the log file for GDMAE self-supervised training?  What is the Chamfer Distance loss approximately when it reaches convergence?

**Comment by Nightmare-n** [comment]
![train_loss](https://user-images.githubusercontent.com/47742823/233530746-d20253fb-7911-4457-8f13-3a0b69d3c9f4.png)
Here is the self-supervised training loss, where the loss seems to converge to 0.3 (30 epochs).
